### PR TITLE
Add logo usage rights explainer

### PR DESCRIPTION
**Pull request by jeanduplessis**
As per [this Slack conversation](https://sourcegraph.slack.com/archives/C01GJFE26BX/p1648881718132679) this update states that we generally provide approval for small startup companies when it comes to using our logo on their website if we use their product/service. Consent can be requested from the primary account contact.

### CODE REVIEW

File: content/departments/marketing/brand/brand_guidelines/logo.md [hunk 1]
```diff
@@ -190,3 +190,9 @@ See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=22
 ### Wordmark use
 
 See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=23)
+
+## Logo usage rights
+
+Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap/#commercial-negotiations)).
```
Comment (jeanduplessis) [inline content/departments/marketing/brand/brand_guidelines/logo.md]:
```suggestion
Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap.md#commercial-negotiations)).
```

File: content/departments/marketing/brand/brand_guidelines/logo.md [hunk 2]
```diff
@@ -190,3 +190,9 @@ See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=22
 ### Wordmark use
 
 See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=23)
+
+## Logo usage rights
+
+Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](/departments/finance/process/ap.md#commercial-negotiations)).
```
Comment (jeanduplessis) [inline content/departments/marketing/brand/brand_guidelines/logo.md]:
```suggestion
Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](../../../finance/process/ap.md#commercial-negotiations)).
```

File: content/departments/marketing/brand/brand_guidelines/logo.md [hunk 3]
```diff
@@ -190,3 +190,9 @@ See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=22
 ### Wordmark use
 
 See [PDF](https://sourcegraphstatic.com/Sourcegraph_Brand_Guidelines.pdf#page=23)
+
+## Logo usage rights
+
+Displaying the Sourcegraph logo on your website or other digital asset is only allowed with express permission and usually governed through the vendor contract (refer to [Sourcegraph's procurement policy](../../../finance/process/ap.md#commercial-negotiations)).
+
+Startup companies, who's product or service we use and who we don't have a formal vendor contact with, that would like to showcase the Sourcegraph logocan request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO and under 200 people employed. Written permission can be obtained from the primary contact for the account.
```
Comment (tammy-zhu) [inline content/departments/marketing/brand/brand_guidelines/logo.md]: replace "who's" with "whose"

Comment (tammy-zhu) [inline content/departments/marketing/brand/brand_guidelines/logo.md]: remove "and who we don't have a formal vendor contract with" 
reason for removal: we should have a vendor contract with every company whose product/service we use, even if it's online terms, so let's avoid condoning using vendor services without a formal vendor contract.

Comment (jeanduplessis) [inline content/departments/marketing/brand/brand_guidelines/logo.md]:
```suggestion
Startup companies, whose product or service we use, that would like to showcase the Sourcegraph logo can request approval as long as their company conforms to the general criteria of a startup: VC-backed, pre-IPO, and under 200 people employed. Written permission can be obtained from the primary contact for the account.
```

**Review by tammy-zhu — APPROVED** [review]

**Pull request merged**